src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -1,35 +1,95 @@
 package seedu.address.logic.parser;
 
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.command.edit.EditCommand.INVALID_FIELDS;
+import static seedu.address.logic.command.edit.EditCommand.MESSAGE_USAGE;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertWanderLustParseEditCommandFailure;
 import static seedu.address.logic.parser.CommandParserTestUtil.assertWanderLustParseEditCommandSuccess;
 import static seedu.address.logic.parser.ParserUtil.MESSAGE_INVALID_INDEX;
 import static seedu.address.testutil.typicals.TypicalIndexes.INDEX_FIRST;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.logic.command.edit.EditAccommodationCommand;
+import seedu.address.logic.command.edit.EditActivityCommand;
 import seedu.address.logic.command.edit.EditDescriptor;
 import seedu.address.logic.command.edit.EditFriendCommand;
+import seedu.address.logic.command.edit.EditTravelPlanCommand;
+import seedu.address.model.commons.Cost;
 import seedu.address.model.commons.Name;
 
 public class EditCommandParserTest {
 
     private EditCommandParser parser = new EditCommandParser();
 
-    //Parse EditFriend
+    //simple edit friend
     @Test
-    public void parse_validArgs_returnsEditCommand() {
+    public void parse_editFriend_success() {
 
         EditDescriptor editDescriptor = new EditDescriptor();
         editDescriptor.setName(new Name("Bobby"));
         assertWanderLustParseEditCommandSuccess(parser, " -friend 1 n/Bobby",
                 new EditFriendCommand(INDEX_FIRST, editDescriptor));
     }
 
-    //Parse EditActivity
     @Test
-    public void parse_invalidArgs_throwsParseException() {
-        assertWanderLustParseEditCommandFailure(parser, " -activity i/2",
+    public void parse_editActivity_success() {
+        EditDescriptor editDescriptor = new EditDescriptor();
+        editDescriptor.setName(new Name("Ice Skating"));
+        assertWanderLustParseEditCommandSuccess(parser, " -activity 1 n/Ice Skating",
```

Comment:
This method name I changed to assertEditParseSuccess(), wait I merge then you update

Reply:
okok